contrib/tracefile-analyser.php

```diff
@@ -62,16 +78,25 @@
 foreach( $functions as $name => $f )
 {
 	$c++;
+
 	if ( $c > $elements )
 	{
 		break;
 	}
-	printf( "%-{$maxLen}s %5d  %3.4f %8d  %3.4f %8d\n",
-		$name, $f['calls'],
-		$f['time-inclusive'], $f['memory-inclusive'],
-		$f['time-own'], $f['memory-own'] );
+
+	printf(
+		"%-{$maxLen}s %5d  %3.4f %8d  %3.4f %8d\n",
+		$name,
+		$f['calls'],
+		$f['time-inclusive'],
+		$f['memory-inclusive'],
+		$f['time-own'], $f['memory-own']
+	);
 }
 
+echo PHP_EOL;
```

Comment:
I'm using ``\n`` at the end of the ``printf`` already, this just adds an extra line before returning to the shell prompt, so I don't think this is needed.